app/Services/TaskActivityService.php

```diff
@@ -23,7 +18,7 @@ public function __construct(TaskComment $taskComment, TaskHistory $taskHistory)
         $this->taskHistory = $taskHistory;
     }
 
-    public function getCollection(string $type,int $taskId): array
+    private function _getCollection(string $type, int $taskId): array
```

Comment:
PHP EA: You shouldn't use underscore as an indicator of *private* function

Reply:
remove it class